post-scan-actions/fss-to-securityhub/README.md

```diff
@@ -0,0 +1,12 @@
+# FSS to Security Hub Project
+
+Pushes FSS malware detections to AWS Security Hub integration.
+
+Zip the `fss-to-securityhub`'s contents and upload it to a bucket and take note of the key.
```

Comment:
The folder node should follow the `provider-language-name` format. Please consider  `aws-nodejs-securityhub-integration`.